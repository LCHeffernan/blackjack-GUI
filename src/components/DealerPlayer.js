import React from "react";

const DealerPlayer = () => <div className="cards-container dealer">dealer</div>;

export default DealerPlayer;
