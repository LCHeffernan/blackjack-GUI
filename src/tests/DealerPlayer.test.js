import React, { render } from "@testing-library/react";
import DealerPlayer from "../components/DealerPlayer";

describe("Dealer", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<DealerPlayer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
