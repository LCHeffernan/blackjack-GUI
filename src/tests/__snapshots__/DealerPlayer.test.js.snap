// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dealer renders correctly 1`] = `
<DocumentFragment>
  <div
    class="cards-container dealer"
  >
    dealer
  </div>
</DocumentFragment>
`;
